smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude.smithy

```diff
@@ -756,3 +756,28 @@ list suppress {
 @unstable
 @trait(selector: "operation")
 structure httpChecksumRequired {}
+
+// Defines supported checksum for an operation's request or response.
```

Comment:
This should use the `///` [documentation](https://awslabs.github.io/smithy/1.0/spec/core/idl.html#grammar-token-documentation_comment) comment.